Automattic/wp-calypso
# `/subscriptions`: Add locale support in route and link generation.

Closes #75703

## Proposed Changes

* Adds locale support in route and link generation.
* Take note that Calypso Portal will only support MAG16.

## Testing Instructions

* Go to `/subscriptions/sites/(ar|pt-br|zh-cn|...)`, it should display sites list in your defined locale.
* Go to `/subscriptions/settings/(ar|pt-br|zh-cn|...)`, it should display the settings list in your defined locale.
* Go to `/subscriptions/(sites|settings)/en`, it should rewrite itself back to `/subscriptions/(sites|settings)`.

## Pre-merge Checklist
